.github/workflows/publish-pacts.yaml

```
@@ -0,0 +1,50 @@
+name: Publish Pacts
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ dev ]
+
+jobs:
+  setup-and-test:
+    runs-on: ubuntu-latest
+    outputs:
+      nb-b64: ${{ steps.encode.outputs.nb-b64 }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install yarn
```

```suggestion
      - name: Install dependencies
```

Nit: this isn't installing yarn. It's install Terra UI dependencies using yarn.